app/models/correctReturn/AddASmallProducer.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.correctReturn
+
+import models.SmallProducer
+import play.api.libs.json._
+
+case class AddASmallProducer(producerName:Option[String], referenceNumber:String, lowBand: Long, highBand: Long)
+
+object AddASmallProducer {
+  implicit val format = Json.format[AddASmallProducer]
+
+  def toSmallProducer(addASmallProducer: AddASmallProducer): SmallProducer =
```

Comment:
also we don't do this in returns, is the validation different?

Reply:
No - it just seems a neater way of converting between models than doing it in the controller